playbook2.yaml

```diff
@@ -0,0 +1,99 @@
+- name: Projex4 - run your To-Do app on the managed VM
+  hosts: managedNodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: TRELLO_BOARD_ID
+      prompt: Please enter Trello Board ID >
+      default: 64ae6f476f946f8c451a5cb2
+      private: false
+
+    - name: TRELLO_TODO_ID
+      prompt: Please enter Trello TODO List ID >
+      default: 64ae6f476f946f8c451a5cb9
+      private: false
+
+    - name: TRELLO_DOIN_ID
+      prompt: Please enter Trello DOIN List ID >
+      default: 64ae6f476f946f8c451a5cba
+      private: false
+
+    - name: TRELLO_DONE_ID
+      prompt: Please enter Trello DONE List ID > 
+      default: 64ae6f476f946f8c451a5cbb
+      private: false
+
+    - name: TRELLO_APIKEY
+      prompt: Please enter Trello API key > 
+      #default: 7d069c46c188afd7a7eaeb18851b9fac #sensitive so 'not' committing to git
+      private: true
+
+    - name: TRELLO_TOKEN
+      prompt: Please enter Trello token >
+      #default: ATTAcff8cd91ef9d8108a296d4954de006c36a5f27507477246e8cd87d76edbdf881F66B1476 #sensitive so 'not' committing to git
+      private: true
+
+  tasks:
+  - name: use the Yum package manager to install Git and Python3
+    ansible.builtin.yum:
+      name:
+        - git
+    become: yes
+  
+  - name: 'Install Python Dependencies: part 3'
+    ansible.builtin.command: sudo yum install libffi-devel bzip2-devel wget openssl11-devel -y --skip-broken
+
+  - name: Add path for Ansible
```

Comment:
This might be slightly confusingly named because it's not the ansible command we're adding to the path!